spec/reek/rake/task_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../spec_helper'
+require_lib 'reek/rake/task'
+
+RSpec.describe Reek::Rake::Task do
+  describe '#initialize' do
+    it 'works' do
+      Reek::Rake::Task.new
+    end
```

Comment:
I think this should either be `it 'can be called without arguments'` (or whatever this test is testing) or dropped altogether. :)